pages/api/collect.js

```diff
@@ -68,14 +68,12 @@ export default async (req, res) => {
     url = url.replace(/\/$/, '');
   }
 
-  const eventUuid = uuid();
-
   if (type === 'pageview') {
-    await savePageView(website, { session, url, referrer });
+    await savePageView(websiteId, { pageViewId: uuid(), session, url, referrer });
```

Comment:
Why is this pageViewId instead of id?

Reply:
Because its not just passing in a pageview data, but session data too.  Just clearly identifies the data amongst the other stuff we're passing in.